app/pages/_app.tsx

```diff
@@ -2,9 +2,25 @@ import "@klimadao/lib/theme/colors.css";
 import "@klimadao/lib/theme/normalize.css";
 import "@klimadao/lib/theme/globals.css";
 import type { AppProps } from "next/app";
+import { useEffect } from "react";
+
+import { I18nProvider } from "@lingui/react";
+import { i18n } from "@lingui/core";
+
+import { init } from "../lib/i18n";
 
 function MyApp({ Component, pageProps }: AppProps) {
-  return <Component {...pageProps} />;
+  useEffect(() => {
+    // Activate the default locale on page load
+    init();
```

Comment:
Does the fact that this only runs after mount cause a flash of un-translated text?

Reply:
I think so. I am a react/nextjs noob so I do not really know the lifecycles etc... Let me know if there is a better place to put init stuff.

Reply:
OK we can fix this in a follow-up PR :)